benchmarks/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
```

Comment:
This is problematic. It needs to match what we do in our repos.

Reply:
Good, exactly what we need to talk about. Can't we just think of this app to be for the Benchmarks Server only, and just ensure it builds correctly with KoreBuild?

Reply:
Looks like we might have to put a dependencies file dynamically that would match the KoreBuild one, probably by getting/building one from the meta packages for a specific ASP.NET version.

Reply:
No we just need to the project to support overrides specified by the benchmarks driver.